.cloudbuild/ci/integration-tests.yaml

```diff
@@ -12,7 +12,7 @@ options:
 steps:
   # Run the integration tests. Actual content of this job depends on the changes
   # detected in the PR
-  - name: quay.io/gravitational/teleport-buildbox:teleport10
+  - name: public.ecr.aws/gravitational/teleport-buildbox:teleport10
```

Comment:
Should we change this to 11 while we're here?

Reply:
Fixed in https://github.com/gravitational/teleport/pull/15058/commits/0f39a01cb8104859d236bbf442a8365991b5b853